library/platform_util.c

```diff
@@ -165,3 +165,46 @@ extern inline void mbedtls_put_unaligned_uint32(void *p, uint32_t x);
 extern inline uint64_t mbedtls_get_unaligned_uint64(const void *p);
 
 extern inline void mbedtls_put_unaligned_uint64(void *p, uint64_t x);
+
+#if defined(MBEDTLS_HAVE_TIME) && !defined(MBEDTLS_PLATFORM_MS_TIME_ALT)
+
+#include <time.h>
+#if !defined(_WIN32) && (defined(unix) || \
+    defined(__unix) || defined(__unix__) || (defined(__APPLE__) && \
+    defined(__MACH__)))
```

Comment:
Hmm, looks like I've been reading this wrong. Now I know what it actually says it's fine, but something like

```
#if !defined(_WIN32) &&
    (defined(unix) || defined(__unix) || defined(__unix__) ||
    (defined(__APPLE__) && defined(__MACH__)))
```

might have avoided my confusion